spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/README.adoc

```diff
@@ -43,3 +43,13 @@ Similarly for `spring.cloud.gcp.credentials-location`, if a property is configur
 https://docs.spring.io/spring/docs/current/spring-framework-reference/html/resources.html#resources-implementations[file system paths, HTTP URLs, etc.]
 If no property is configured, the provided `CredentialsProvider` returns the
 http://google.github.io/google-auth-library-java/releases/0.7.1/apidocs/com/google/auth/oauth2/GoogleCredentials.html#getApplicationDefault()[application default credentials].
+
+If your app is running on Google App Engine or Google Compute Engine, you should omit the
```

Comment:
Again,
"If your app is running on Google App Engine or Google Compute Engine, in most cases, you will want to omit the `spring.cloud.gcp.credentials-location` property, and instead, let the Spring Cloud GCP Core Starter get the built-in credentials in those environments."